main/src/ca/uwaterloo/flix/language/phase/MonoDefs.scala

```diff
@@ -79,46 +79,40 @@ object MonoDefs {
       *
       * NB: Applies the substitution first, then replaces every type variable with the unit type.
```

Comment:
I guess this documentation and the documentation for `StrictSubstitution` is no longer accurate.